src/components/AddPaymentMethodMenu.js

```diff
@@ -19,7 +20,7 @@ const propTypes = {
     }),
 
     /** Username for PayPal.Me */
-    payPalMeUsername: PropTypes.string,
+    payPalMeUserDetails: bankAccountPropTypes,
```

Comment:
Maybe we should create a separate prop type for PayPal data

Reply:
+1

Reply:
The onyx has the paypal details, which includes more that the username

Reply:
+1, it's gonna be a duplicate anyways